packages/property-grid/src/PropertyGridManager.ts

```diff
@@ -9,24 +9,36 @@ import { I18N } from "@bentley/imodeljs-i18n";
 import { getClassName, UiError } from "@bentley/ui-abstract";
 import { StateManager } from "@bentley/ui-framework";
 
+/** Supported Feature Flags for  */
+export interface PropertyGridMangerFeatureFlags {
+  /** If true, enables property category group nesting  */
+  enablePropertyGroupNesting?: boolean;
+}
+
 /**
  * Entry point for static initialization required by various components used in the package.
  * @public
  */
 export class PropertyGridManager {
   private static _i18n?: I18N;
-
+  private static _featureFlags: PropertyGridMangerFeatureFlags = {
```

Comment:
maybe add a comment, about this being the _featureFlags obj here being the default values